### ISSUE TITLE
Two handed gun handling?

### ISSUE THREAD

**Issue by Aesop112** [KozGit/DOOM-3-BFG-VR#147]
It's really just so much better in vr holding guns this way. Please. Add this.

**Comment by Jake762x51** [comment]
+1

**Comment by KozGit** [comment]
I'm assuming you mean using both hands to aim the long guns, and not dual wielding?

**Comment by Jake762x51** [comment]
Yes using both hands to aim the long guns, just like in H3VR and Onward.

**Comment by Aesop112** [comment]
Please

**Comment by blakeah** [comment]
+1

**Comment by jdawgzim** [comment]
Then duct-tape flash light to gun?  How would flashlight be handled?

Some games allow one handed firing but then you get more accuracy when you wield a rifle two handed.  This sounds like a difficult change, though. I'm fine with how the game works now.  This is more of a super soldier action game and less of a military shooter.  This is more Serious Sam then Call Of Duty.

**Comment by blakeah** [comment]
While I agree with your reasoning, I feel like more options are for the better!  Currently, the the mod allows you to mount the flash-light to your head which I find works great, and doesn't ruin immersion at all.  What does ruin it however is the fact that when you have a custom 3D printed gun-stock for games such as Onward or H3VR and use it in this Doom3VR, your "off-hand" essentially is floating in space in front of (or above) your gun and can slightly break immersion.  Therefore attaching the off hand to the weapon animations or removing it and allowing the PDA to be static would be a great alternative in this case.

**Comment by Aesop112** [comment]
It's a horror game, and isn't in the flavor of serious sam at all with waves of hordes. It's generally more atmospheric and I think gripping your gun with a second hand would add to the immersion.

**Comment by djsolidsnake86** [comment]
This will be implemented? Doom3 quest 2 port has this
